spec/factories/notes.rb

```diff
@@ -1,9 +1,17 @@
 # frozen_string_literal: true
 
+note_categories = [
+  'note',
```

Comment:
this is now `general` after our slack discussion this morning.

Reply:
Ah but I think this is where the merge conflict is so will get resolved when the merge conflict gets resolved.